specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPackQueries_API.json

```diff
@@ -0,0 +1,464 @@
+          },
+          {
+            "$ref": "#/parameters/MaxItemCountParameter"
+          },
+          {
+            "$ref": "#/parameters/IncludeContentParameter"
+          },
+          {
+            "$ref": "#/parameters/ContinuationTokenParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "A list containing 0 or more queries associated with the Log Analytics QueryPack.",
+            "schema": {
+              "$ref": "#/definitions/LogAnalyticsQueryPackQueryListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "QueryList": {
+            "$ref": "./examples/QueryPackQueriesList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/QueryPacks/{queryPackName}/queries/search": {
+      "post": {
+        "description": "Search a list of Queries defined within a Log Analytics QueryPack according to given search properties.",
+        "operationId": "Queries_Search",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/QueryPackNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MaxItemCountParameter"
+          },
+          {
+            "$ref": "#/parameters/IncludeContentParameter"
+          },
+          {
+            "$ref": "#/parameters/ContinuationTokenParameter"
+          },
+          {
+            "name": "QuerySearchProperties",
+            "description": "Properties by which to search queries in the given Log Analytics QueryPack.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LogAnalyticsQueryPackQuerySearchProperties"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "A list containing 0 or more queries associated with the Log Analytics QueryPack.",
+            "schema": {
+              "$ref": "#/definitions/LogAnalyticsQueryPackQueryListResult"
```

Comment:
>LogAnalyticsQueryPackQueryListResult [](start = 37, length = 36)

any operation returning this definition should be marked as x-ms-pageable ([https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable))

Reply:
done